src/usermod.c

```diff
@@ -1062,7 +1062,8 @@ process_flags(int argc, char **argv)
 				}
 				dflg = true;
 				user_newhome = optarg;
-				if (user_newhome[0] != '/') {
+				/* If no empty string, check whether the path is absolute */
+				if ((user_newhome[0] != '\0') && (user_newhome[0] != '/')) {
```

Comment:
```suggestion
				if ((strlen(user_newhome) > 0) && (user_newhome[0] != '/')) {
```
I think this is better to read, but it is more costly than checking `user_newhome[0] != '\0'`.

Reply:
Please use `!streq(user_home, "")`.  We have it available with `#include "string/strcmp/streq.h"`.

Reply:
Also, since `/` is a more common case, it should probably go first.  Both for optimization reasons (although I care 0 about optimizing this), and for readability reasons (the brain --at least in my case-- cares more about the first condition, and only cares about the second secondarily; just like the compiler).